BUILDING.md

```diff
@@ -60,12 +60,23 @@ Node.js relies on V8 and libuv. We adopt a subset of their supported platforms.
 There are three support tiers:
 
 * **Tier 1**: Full test coverage and maintenance by the Node.js core team and
-  the broader community.
+  the broader community. All commits to the Node.js repository are tested on
+  multiple variants of these platforms. Breakages are considered critical for
+  the purpose of creating releases.
 * **Tier 2**: Full test coverage. Limited maintenance, often provided by the
-  vendor of the platform.
-* **Experimental**: May not compile or test suite may not pass.
-  These are often approaching Tier 2 support but are not quite ready.
-  There is at least one individual providing maintenance.
+  vendor of the platform. All commits to the Node.js repository are generally
```

Comment:
Remove `generally`? If the statement is true without "generally", cool. If not...I'm not sure what "generally" means here. If it just means that the statement isn't strictly true, then write the statement so that it's strictly true. (Maybe "usually" instead of "generally" and drop "All" at the start of the sentence?)

Reply:
explained later in the sentence:
> is not considered essential if there are temporary problems with the test infrastructure.

I'll try and be more clear but tbh getting rid of ambiguity with these is tricky because I'm inventing policy on the fly, they're pretty vague now and are more emergent ideas than anything concrete.